spring-boot-autoconfigure-data-aerospike/src/main/java/org/springframework/boot/autoconfigure/data/aerospike/AerospikeDataProperties.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.boot.autoconfigure.data.aerospike;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * Configuration properties for Spring Data Aerospike.
+ *
+ * @author Igor Ermolenko
+ */
+@ConfigurationProperties(prefix = "spring.data.aerospike")
+public class AerospikeDataProperties {
+    private String namespace;
```

Comment:
```
public class AerospikeConfigurationProperties {

    @NotNull
    private String hosts;
    @NotNull
    private String namespace;

    private String user;

    private String password;

    private String clusterName;

    private int eventLoopsSize = getDefaultEventLoopsSize();
    private int maxConnectionsPerNode = 300;
    private int connectionPoolsPerNode = 1;
    private int maxSocketIdleSec = 55;
    private int tendIntervalMs = 1_000;

    @Min(0)
    private int connectTimeoutMs = 5_000;
    @Min(0)
    private int operationTimeoutMs = 200;

    private boolean sendKey = false;

    private static int getDefaultEventLoopsSize() {
        try {
            return Runtime.getRuntime().availableProcessors();
        } catch (Exception e) {
            return 1;
        }
    }
}
```

Reply:
Reimplemented. Properties **host** and **port** replaced with property **hosts**.